### PR TITLE
Aifc rules

### PR DESCRIPTION
**Pull request by gstoehld**
Business rules are now written in readable code and compiled to CertLogic with a build script

### CODE REVIEW

File: ch-covidcertificate-backend-verifier/ch-covidcertificate-backend-verifier-valuesets/src/main/resources/test_rules.sh [hunk 1]
```diff
@@ -1,15 +1,36 @@
 #!/bin/bash
-
 #Test master rules against test cases in EU repo
 
-cd $(dirname "$0")/../../../
+if [ "$(uname -s)" == Linux ]; then
+  AIFC=./aifc-bin/aifc_linux_x86_64
+else
+  AIFC=./aifc-bin/aifc_linux_x86_64
```
Comment (ubamrein) [inline ch-covidcertificate-backend-verifier/ch-covidcertificate-backend-verifier-valuesets/src/main/resources/test_rules.sh:7]: one should be mac ? :)

File: ch-covidcertificate-backend-verifier/ch-covidcertificate-backend-verifier-valuesets/src/main/resources/verification-rules/TR-CH-0001.aifc [hunk 2]
```diff
@@ -0,0 +1,19 @@
+/*
+For rapid or PCR test, the result must be negative (\"not detected\").
+*/
+if (
+  /* 
+    TODO: fix correct test type description
```
Comment (ubamrein) [inline ch-covidcertificate-backend-verifier/ch-covidcertificate-backend-verifier-valuesets/src/main/resources/verification-rules/TR-CH-0001.aifc:6]: We should fix the todo here :)

File: ch-covidcertificate-backend-verifier/ch-covidcertificate-backend-verifier-valuesets/src/main/resources/verification-rules/VR-CH-0002.aifc [hunk 3]
```diff
@@ -0,0 +1,28 @@
+/*
+Only vaccines in the allowed valueset that have been approved by the EMA or are otherwise accepted in Switzerland are allowed.
+*/
+if (payload.v.0) {
+  /*
+    mp: Medical Product => specifies vaccines
+  */
+  payload.v.0.mp in [
+      /* TODO: vaccine name */
```
Comment (ubamrein) [inline ch-covidcertificate-backend-verifier/ch-covidcertificate-backend-verifier-valuesets/src/main/resources/verification-rules/VR-CH-0002.aifc:9]: same here, either we should remove the todo or add the common vaccine name

**Review by ubamrein — COMMENTED** [review]
We should fix the todos in the rules before merging :)

**Review by ubamrein — APPROVED** [review]

**Review by benz-ubique — APPROVED** [review]

**Comment by delixfe** [comment]
Could you add a reference to AIFC in the docs?

**Comment by gstoehld** [comment]
@delixfe I added a short explanation and link to the readme. The linked repo has more details on the format.

**Review by UBaggeler — APPROVED** [review]

**Pull request merged**